siatune/tune/spaces/choice.py

```diff
@@ -14,16 +14,20 @@ class Choice(BaseSpace):
     """Sample a categorical value.
 
     Args:
-        categories (Sequence): The categories.
-        alias (Sequence, optional): A alias to be expressed.
-            Defaults to None.
+        categories (Sequence | dict): The categories. If categories is dict,
```

Comment:
How about describing it as 'The categorical search space to choice one'?

Reply:
I got it.